api/tests/unit/application/certification-centers/index_test.js

```diff
@@ -45,6 +40,10 @@ describe('Unit | Router | certification-center-router', () => {
     });
 
     it('should accept a string array of one element as division filter ', async () => {
+      // given
+      sinon.stub(certificationCenterController, 'getStudents').callsFake((request, h) => h.response().code(200)); const httpTestServer = new HttpTestServer();
```

Comment:
```suggestion
      sinon.stub(certificationCenterController, 'getStudents').callsFake((request, h) => h.response().code(200)); 
      const httpTestServer = new HttpTestServer();
```

** Nitpick :** Ajout d'un saut de ligne avant la déclaration du server.